Dockerfile

```diff
@@ -10,6 +10,12 @@ RUN apt-get -qq update && apt-get -qq -y install curl libncurses5 libsodium-dev
 RUN mkdir -p /home/aeternity/node
 COPY ./docker/aeternity.yaml /home/aeternity/aeternity.yaml
 
+# Set build git revision
+RUN mkdir /home/aeternity/node/ae_mdw
+COPY .git .git
+RUN BUILD_REV="$(git log -1 --format=%h)" && echo $BUILD_REV > /home/aeternity/node/ae_mdw/AEMDW_REVISION
```

Comment:
That would be nice to have it in Makefile (`make compile`)

Reply:
Yes, with `mix compile` during the docker build the `.git` doesn't exist so it falls in the else and reads it from the file. Is this good for you @venimus?

Reply:
good enough